src/views/Nav.js

```diff
@@ -132,7 +132,7 @@ export const Nav = () => {
           </NavLink>
         </li>
         <li>
-          <NavLink activeClassName="selected" to={`/${user?.handle}`}>
```

Comment:
We need to retain the URL here so that people can externally link to the user profile via the handle.